### ISSUE TITLE
There's still nothing in this repo

### ISSUE THREAD

**Issue by CyberFlameGO** [Bios-Marcel/Bill#1]
Is this repo a dead end?

**Comment by Bios-Marcel** [comment]
Funny thing. I made this repository, when I decided that I want to make something related to Discord.
I created this repository on January 4th and on January 10th I decided to make https://github.com/Bios-Marcel/cordless. In the end, I just decided that Bill wasn't worth the work, as it was just a project that I wanted to use in order to learn something, but there's nothing I hate more than making something useless. Someone else in the community I was in, was actually aiming to make a TUI discord client called `cordless`. It's this https://github.com/Southclaws/cordless-old, which he had dropped like 4-5 months before. Yup, that's the story to cordless and the reason this repo is empty :D

**Comment by CyberFlameGO** [comment]
Right. Never knew the history behind cordless so thank-you! Also, are you planning to make Bill into anything anytime soon?

**Comment by Bios-Marcel** [comment]
unlikely
